tests/validation/tests/v3_api/test_configmaps.py

```diff
@@ -0,0 +1,489 @@
+    # using config map
+
+    user_token = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+    project, ns = create_project_and_ns(user_token, namespace["cluster"],
+                                        random_test_name("rbac-cluster-mem"))
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_edit(p_client, ns)
+
+
+def test_rbac_cmap_cluster_member_delete():
+
+    # Verify cluster member can create config map and deploy workload
+    # using config map
+
+    user_token = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+    project, ns = create_project_and_ns(user_token, namespace["cluster"],
+                                        random_test_name("rbac-cluster-mem"))
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_delete(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_owner_create():
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_create(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_owner_edit():
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_edit(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_owner_delete():
+    user_token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_delete(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_member_create():
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_create(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_member_edit():
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_edit(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_member_delete():
+    user_token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    project = rbac_get_project()
+    ns = rbac_get_namespace()
+    p_client = get_project_client_for_token(project, user_token)
+    rbac_configmap_delete(p_client, ns)
+
+
+@if_test_rbac
+def test_rbac_cmap_project_readonly_member_create(remove_resource):
+
+    cluster_owner_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    project, ns = create_project_and_ns(cluster_owner_token,
+                                        namespace["cluster"],
+                                        random_test_name("rbac-test"))
```

Comment:
the user returned from `rbac_get_user_token_by_role(PROJECT_READ_ONLY)` is the read-only user of the project returned from `rbac_get_project()`, so we have no need to create a new project.
And there is a namespace in the project and can be retrieved via `rbac_get_namespace()`

Reply:
Done